src/ui/public/notify/notifier.js

```diff
@@ -310,48 +320,69 @@ Notifier.prototype._showFatal = function (err) {
  * @param  {Error|String} err
  * @param  {Function} cb
  */
-Notifier.prototype.error = function (err, cb) {
-  return add({
+Notifier.prototype.error = function (err, opts, cb) {
+  const overrideable = ['lifetime', 'icon'];
```

Comment:
Yeah, moving up to the module scope probably makes sense here, since the rules are unlikely to change from method to method. Good call. And `overrideableOptions` would make a better name.